openapi/v2/core-metadata.yaml

```diff
@@ -426,10 +389,13 @@ components:
           type: string
         name:
           type: string
+        isHidden:
+          type: boolean
+          description: "Indicate the visibility of any resource via Core Command."
```

Comment:
Change `any resource via Core Command` to `the DeviceResource via a CoreCommand`

Reply:
Modified.